dev-docs/RFCs/v7.x/gpu-grid-layer.md

```diff
@@ -0,0 +1,40 @@
+# RFC: GPU Acceleration in GridLayer
+
+* **Authors**: Ravi Akkenapally
+* **Date**: May 2019
+* **Status**: **Draft**
+
+
+## Abstract
+
+GPU accelerated aggregation is currently available in `ScreenGridLayer` and `ContourLayer`. In 6.3 release, `GPUAggregator` is enhanced to support aggregation of multiple weights required for `GridLayer`. This RFC proposes adding GPU accelerated aggregation to `GridLayer`.
+
+
+## Fallback cases
+
+There are several features of `GridLayer` that can't be supported by GPU Aggregation.
+
+* `Lower and higher Percentile` : When these props are set to a non zero value, grid cells are sorted based on their aggregated values and some of the begining and ending cells are ignored (not rendered) based on specified percentile value. On GPU, sorting is not possible.
+
+* `Aggregation operation` : Any aggregation other than `Min`, `Max`, `SUM` or `MEAN` is not supported on GPU.
```

Comment:
Are you doing this with blends? If so you could add "product" to the list: `gl.blendFunc(gl.ZERO, gl.SRC_COLOR)`, if that would be useful.

Reply:
Yes, GPUAggregator uses blending, I will add `Product` to feature and address it in a different PR.